src/frontend/src/flows/register.ts

```diff
@@ -21,12 +21,12 @@ const pageContent = html`
   <div class="l-container c-card c-card--highlight">
     <hgroup>
       <div class="c-logo">${icLogo}</div>
-      <p class="t-lead">What device are you using?</p>
+      <p class="t-lead t-centered">What device are you using?</p>
     </hgroup>
     <form id="registerForm" class="l-stack">
       <input
         id="registerAlias"
-        placeholder="Device name"
+        placeholder="Chrome on iPhone"
```

Comment:
This is definitely a better placeholder than what we had before. But this browser separation of webauthn devices is an Apple think. So this might confuse users on Windows / Linux where including the browser name in the device name does not make sense and registering the device using different browsers will fail (due to the webauthn device already being regitstered). How about we suggest just `iPhone` for now?

Reply:
Reverted to what we had previously, we'll need to work on this